container/CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Changelog
```

Comment:
This seems like a new separate file. I am guessing because we keep it separate for container. 

Would be nice to have a small comment here like in main [changelog](https://github.com/SAP/luigi/blob/main/CHANGELOG.md?plain=1#L3-L25)